src/cli/onefuzz/api.py

```diff
@@ -170,31 +170,34 @@ class Files(Endpoint):
     endpoint = "files"
 
     @cached(ttl=ONE_HOUR_IN_SECONDS)
-    def _get_client(self, container: str) -> ContainerWrapper:
+    def _get_client(self, container: primitives.Container) -> ContainerWrapper:
```

Comment:
Style suggestion, maybe just import `Container` into this module so we don't have to keep qualifying it.

Reply:
I guess my suggestion includes the primitives we use below, too.

Reply:
We previously did this, but changed to avoid conflict with the class names for the API endpoints (such as `Containers`)

Reply:
Leaving for now, we can revisit that in the future, but it would be a larger impact.